### DISCUSSION TITLE
mean function returns dimensionless quantities

### DISCUSSION THREAD

**Discussion by mattyd100** [hgrecco/pint#2096] [Q&A] [answered]
When I calculate the mean of a quantity (such as multiple length measurements), a dimensionless value is returned. Shouldn't it have units? Or am I missing something?

`import numpy as np
import pint

ureg = pint.UnitRegistry()
length = np.array([1, 2, 3]) * ureg('in')

print(statistics.mean(length))`

**Comment by andrewgsavage** [comment, marked as answer]
the statistics doesn't have support for pint. you can use np.mean instead
